packages/kbn-some-dev-log/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/some-dev-log
```

Comment:
Do we want to move this into `mdx` and add it to our operations docs?

Reply:
Done in [`31f8b09` (#136207)](https://github.com/elastic/kibana/pull/136207/commits/31f8b095d6a98f9a07290fcfa6a4869b805f91c5)